### ISSUE TITLE
All is working but why.

### ISSUE THREAD

**Issue by Gameadictive** [renzuzu/renzu_garage#305]
![server lua-garages1](https://user-images.githubusercontent.com/65845786/200836681-d883d9e3-25bf-4bda-990c-669e2a11ad7a.PNG)
![server lua-garages](https://user-images.githubusercontent.com/65845786/200836686-200b0692-d8bd-4afc-b617-1124ba2354a4.PNG)

**Comment by renzuzu** [comment]
Thia is not a main branch?

Server.lua is not 1000 lines

**Comment by Gameadictive** [comment]
No, is not the main branch, server.lua from the last version is 1300 lines.
